commit 6911c0275e81ea844355e8b24129dd44457d0f41 (refs/changes/88/3188/1)
Author: Jared Morrow <jm@joyent.com>
Date:   2018-01-10T12:37:33-07:00 (1 year, 9 months ago)
    
    MANTA-3553 Muppet should recover when Zookeeper returns after being down

diff --git a/muppet.js b/muppet.js
index 6ab8d19..44c0b53 100644
--- a/muppet.js
+++ b/muppet.js
@@ -286,8 +286,7 @@ function zookeeper(cfg) {
      * exhaustion it will emit a `failed` event.
      */
     zk_client.on('failed', function onFailed(err) {
-        cfg.log.fatal(err, 'ZooKeeper: received failed event');
-        process.exit(1);
+        cfg.log.error(err, 'ZooKeeper: received failed event');
     });
 }
 
